packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -131,6 +131,10 @@ export interface ISemanticColors {
    * Background for success
    */
   successBackground: string;
+  /**
+   * The color for signals
+   */
+  signal: string;
```

Comment:
I'm not sure this makes sense as a semantic slot. It is extremely specific. @phkuo Is there an alternative?